src/plugins/usage_collection/common/types/stats/core_metrics_duplicated.ts

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Overall this makes sense, it is giving us the full types of the endpoint and we will be able to detect when our response changes👌🏻

Nit: I would name this file "core_metrics" then add a doc comment explaining this starts as a duplicate of the core ops metrics.